Formula/sip.rb

```diff
@@ -15,6 +15,8 @@ class Sip < Formula
     sha256 cellar: :any_skip_relocation, mojave:        "7b5c3b19640a10879a1c62b7be2be271fe55dd0139d0af9a7deed5ffe18fdf0f"
   end
 
+  disable! date: "2021-06-16", because: "sip installs unexpected dependencies"
```

Comment:
You mean it installs components when you run it?

Reply:
`sip` installs `pyparsing`, `packaging` and `toml`, and `sip` is not only a command line application, other projects need `sip` libraries.

Reply:
We can't disable `sip`. This is needed by `qscintilla2`. Also, a lot of users use it, and it's caused zero complaints:

==> Analytics
==> install (365 days)
Index | Name (with options)                                                                                    |   Count |  Percent
-----:|--------------------------------------------------------------------------------------------------------|--------:|--------:
1     | sip                                                                                                    | 270,675 |  100.00%
2     | sip --with-python3                                                                                     |       5 |    0.00%
3     | sip --HEAD                                                                                             |       4 |    0.00%
Total |                                                                                                        | 270,684 |  100.00%


==> install-on-request (365 days)
Index | Name (with options)                                                                                     |  Count |  Percent
-----:|---------------------------------------------------------------------------------------------------------|-------:|--------:
1     | sip                                                                                                     | 39,138 |   99.99%
2     | sip --HEAD                                                                                              |      4 |    0.01%
Total |                                                                                                         | 39,142 |  100.00%

Reply:
Thoughts here, @Bo98?

Reply:
It would be a legacy of history before `pyqt` 6.x, `sip` is build dependency but in 5.x it was specified as normal dependency.